saleor/graphql/schema.graphql

```diff
@@ -2407,6 +2471,12 @@ type Mutation {
   permissionGroupDelete(id: ID!): PermissionGroupDelete
   permissionGroupAssignUsers(id: ID!, users: [ID!]!): PermissionGroupAssignUsers
   permissionGroupUnassignUsers(id: ID!, users: [ID!]!): PermissionGroupUnassignUsers
+  invoiceCreate(orderID: ID!, input: InvoiceInput): InvoiceCreate
+  invoiceDelete(orderID: ID!, invoiceID: ID!): InvoiceDelete
```

Comment:
Why we need `orderID` here? Just the invoice ID should be enough:
```suggestion
  invoiceDelete(id: ID!): InvoiceDelete
```

Reply:
The same question for `requestInvoiceDelete`.

Reply:
@patrys explicitly asked for this to add additional level of security I guess.

Reply:
I'd consider calling them something like `orderCreateInvoice` as it's really an operation on the order.